layers/b2qt/recipes-qt/automotive/neptune3-ui/pelux.conf

```diff
@@ -0,0 +1,6 @@
+[Service]
+ExecStart=
+ExecStart=/opt/neptune3/neptune3-ui -r --dbus session -c am-config-neptune.yaml -platform eglfs $EXTRA_ARGUMENTS
+WorkingDirectory=/opt/neptune3
```

Comment:
Is this needed? Already present in https://code.qt.io/cgit/yocto/meta-boot2qt.git/tree/meta-boot2qt/recipes-qt/automotive/neptune3-ui/neptune.service?h=thud

Reply:
I also wonder if we should replace
`Environment=AM_STARTUP_TIMER=1`
with
`Environment=AM_STARTUP_TIMER=/tmp/am-startup-timing.log`